Makefile

```diff
@@ -0,0 +1,17 @@
+up:
+	docker-compose up -d --remove-orphans
+
+down:
+	docker-compose down
+
+clean:
+	docker-compose down --volumes --remove-orphans
+
+update:
+	docker-compose pull
+	sleep 2
+	docker-compose up -d --force-recreate
+	-docker rmi $$(docker images -aq -f 'dangling=true')
```

Comment:
Does `docker image prune` accomplish the same thing?

Reply:
hmm seems like `docker image prune` is the new way of doing this...